data/elan-2cf7.tablet

```diff
@@ -0,0 +1,13 @@
+[Device]
```

Comment:
The CI fails because of the missing `sysinfo` links in the data files. Having those makes reproducing in the future easier when we need to figure out why a device doesn't work. See e.g. bf45744ad6421aff840ba4a2558e4285db93269a as a reference (from #492)